### DISCUSSION TITLE
How to change ConEmu font ?

### DISCUSSION THREAD

**Discussion by akof1314** [gitextensions/gitextensions#9742] [Q&A] [answered]
I want to change the font style to Microsoft Yahei

![image](https://user-images.githubusercontent.com/2639552/142592017-3b2050f3-709c-4b7b-a1ea-aa29a06c8682.png)

**Comment by pmiossec** [comment, marked as answer]
It is not possible at the moment.
It has been made in #9332 but it has not been released at the moment.
You will have to wait for the next version to be released.

**Reply by akof1314** [reply]
thank you
